tools/topology/topology2/cavs-sdw.conf

```diff
@@ -79,6 +79,7 @@ Define {
 	SDW_JACK_CAPTURE_CH 2
```

Comment:
@singalsu Can you add the cherry-picked line to commit ('git cherry-pick -x' does this automatically)? Something like "(cherry picked from commit 3da8e6474531411bef64819113a29a7edbd51fcf)". See "git log origin/stable-v2.10" for examples.

Reply:
It's not yet merged to git main from https://github.com/thesofproject/sof/pull/9690 . Should I make a separate PR for this?

Reply:
@singalsu  #9690 now merged, can you update this one?